[pull] primary from flybywiresim:primary (#135)
Valastiri/fbw-userguide: docs/beginner-guide/preparing-mcdu.md

![mcdu1b](../assets/beginner-guide/mcdu/mcdu1b.png)

> Now continue setting up the MCDU as described above beginning at [**^^F^^**LIGHT PLAN](#flight-plan) above.

Reviewer: ```suggestion
Now continue setting up the MCDU as described in the [**^^F^^**LIGHT PLAN](#flight-plan) section above.
```
